aiida_quantumespresso/parsers/pw.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, calc):
 
         :param calculation: instance of the PwCalculation
         """
+        self._require_xml = calc._require_xml if hasattr(calc, "_require_xml") else True
```

Comment:
I don't quite understand why we should define `_require_xml` in the `Calculation` class, if it is only used in the parser?
Should we not just define it in the parser class, and optionally allow the user to override it with a parser option in the `settings` input node?

Reply:
I only put it here because I couldn't work out how to do what you say! My goal was to make sure the standard behaviour of standard `PwCalculation`s (requiring the xml) wasn't changed, hence setting it in the `Calculation` class. But if we can use the `settings` node that makes more sense